ansible/roles/alertmanager/tasks/main.yml

```diff
@@ -19,6 +21,7 @@
       max-size: 50m
       max-file: "3"
     labels:
+      maintainer: "The Prometheus Authors <prometheus-developers@googlegroups.com>"
```

Comment:
y tho

Reply:
![image](https://user-images.githubusercontent.com/176002/92139070-86b2b400-ee0f-11ea-9154-cb3e6113b524.png)

e4c85a0 fixes it